src/HorseProfile/HorseProfile.js

```diff
@@ -1,6 +1,93 @@
+import React from "react";
+import { useParams, Link } from "react-router-dom";
+import mockData from "../mockData";
+
 function HorseProfile() {
+  const horseId = Number(useParams().id);
+  let horse;
```

Comment:
Using let and const for horse, owner, vet and farrier definitely works for now, but we may need to use state later on if we add an 'edit' button to this page.